frontend/js/modules/group.js

```diff
@@ -351,7 +352,16 @@ module.exports = (function () {
   };
 
   view.group = function (c, g) {
+    console.log(u());
+    
     var padRoute     = '/mypads/group/' + g._id;
+    if(u().watchlist!= null){
```

Comment:
Cache the result of u() in a variable.  And then you can declare isWatched using a ternary operator, instead of needing to use an if-else